### ISSUE TITLE
Compilation fails with capstone 3.0.4

### ISSUE THREAD

**Issue by lenerd** [radare/radare2#7469]
Hello,
when I try to compile the current master (1c0638c9594c7eaba481255f7eab0aff6ee38219) using
```
./configure --prefix=/usr --with-syscapstone --with-syszip --with-openssl
make
```
it fails with the following error:
```
/home/lennart/git/radare2/libr/..//libr/anal/p/anal_arm_cs.c:1272:7: error: ‘ARM64_INS_NEGS’ undeclared (first use in this function)
  case ARM64_INS_NEGS:
       ^~~~~~~~~~~~~~
```
The current stable version of capstone, 3.0.4, does not define the identifier in `capstone/arm64.h`. It is however available in capstone-next.

**Comment by Maijin** [comment]
we are using capstone next

**Comment by radare** [comment]
Dont use --with-syscapstone

Also, we may support cs3 too. So thats a regression and this case must be ifdefed. Can you send a pr?

**Comment by lenerd** [comment]
This was fixed in a58e556c343f0e96bbc310c1aac19de637bce054. Thank you!
